### PR TITLE
feat!: drop support for current required behavior and just provide the conversion follow specs

### PR DESCRIPTION
**Pull request by LumaKernel**
## Types of changes

- [x] Bug fixes
  - resolves https://github.com/aspida/openapi2aspida/issues/181
- [x] Features
  - resolves https://github.com/aspida/openapi2aspida/issues/132, resolves https://github.com/aspida/openapi2aspida/issues/138
- [ ] Maintenance
- [ ] Documentation

## Changes

- ...

## Additional context

This is breaking change but not bump the major version.

## Community note

Please upvode with reacting as :+1: to express your agreement.

### CODE REVIEW

File: src/builderUtils/props2String.ts [hunk 1]
```diff
@@ -41,10 +41,14 @@ const values2String = (values: PropValue[], hasOf: PropValue['hasOf'], indent: s
 
 const isMultiLine = (values: PropValue[]) => values.find(v => !v.isEnum && Array.isArray(v.value))
 
+const escapeDecription = (desc: string): string => {
```
Comment (LumaKernel) [inline src/builderUtils/props2String.ts:44]: https://github.com/aspida/openapi2aspida/issues/181

**Comment by LumaKernel** [comment]
note: freee publishes newer version of their schema: https://github.com/freee/freee-api-schema

**Review by solufa — APPROVED** [review]
この破壊的な変更提案に対し、オリジナル開発者として同意しました

**Review by hashue — APPROVED** [review]

**Comment by hashue** [comment]
私が確認した限りだと怪しいところはないと思います。そのため、私もこの破壊的変更に同意します

**Pull request merged**